ui/rgb/src/lib.rs

```diff
@@ -101,6 +101,7 @@ impl Argb {
     /// Outer-ring color during user QR scans
     pub const PEARL_RING_USER_QR_SCAN: Argb = Argb(None, 30, 20, 0);
     pub const PEARL_RING_USER_QR_SCAN_SPINNER: Argb = Argb(None, 28, 25, 10);
+    pub const PEARL_RING_USER_QR_SCAN_WAVE: Argb = Argb(None, 255, 255, 255);
```

Comment:
this is way too bright. Thomas M asked for 25% reduction based on value of 40. So I changed to 30 in https://github.com/worldcoin/orb-software/pull/401